.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
It's weird that it didn't break anything before, because since the first commit Node v0.10 wasn't supported